Please update to latest version of redux-saga

With latest version we have error: "Error: Cannot find module 'redux-saga/lib/effects'", because effects now should be imported from 'redux-saga/effects';

--- a/lib/utils/effects.js
+++ b/lib/utils/effects.js
@@ -7,7 +7,14 @@ function knowEffects() {
     return miscEffects.includes(effect)
   })
 
-  return Object.keys(require("redux-saga/lib/effects")).concat(other)
+  var reduxSagaEffects
+  try {
+    reduxSagaEffects = require("redux-saga/lib/effects")
+  } catch (err) {
+    reduxSagaEffects = require("redux-saga/effects")
+  }
+
+  return Object.keys(reduxSagaEffects).concat(other)
 }
 
 function isEffectImport(value) {

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
   },
   "peerDependencies": {
     "eslint": ">= 4.1.0",
-    "redux-saga": ">= 0.11.1 < 1"
+    "redux-saga": ">= 0.11.1 < 1 || 1.0.0-beta.3"
   },
   "devDependencies": {
     "eslint": "4.19.1",

